src/System.Private.ServiceModel/src/System/ServiceModel/Description/XmlSerializerOperationBehavior.cs

```diff
@@ -412,6 +412,12 @@ private void CreateHeaderDescriptionTable(MessageDescription message, MessageInf
                         else if (headersMapping != null)
                         {
                             XmlMemberMapping memberMapping = headersMapping[headerNameIndex++];
+#if FEATURE_NETNATIVE
```

Comment:
We have a lot of #if blocks that we need to eventually remove. The only technical reason for an #if block is if the code won't compile if left in. Could we do a runtime check instead to detect we've been compiled to native to skip the checks?